backend/config.py

```diff
@@ -27,44 +27,51 @@ class Config:
     )
     SQLALCHEMY_ENGINE_OPTIONS = {"pool_recycle": 299}
     SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    """jwt config"""
     JWT_BLACKLIST_ENABLED = True
 
     """mail config"""
-    MAIL_DEBUG = True  # always on, needed for error checking
-    MAIL_USE_TLS = True  # always on, for security (TM)
-    MAIL_USE_SSL = False  # always off, for security (TM)
-    MAIL_WEB_SERVER = ""  # defaults to empty str, might be set later
+    MAIL_DEBUG = True  # always on, for error checking
+    MAIL_USE_TLS = True  # always on, for security
+    MAIL_USE_SSL = False  # always off, for security
+    MAIL_WEB_URL = ""  # used only for ethereal.mail
 
-    MAIL_SERVER = os.environ["MAIL_SERVER"]
-    MAIL_PORT = os.environ["MAIL_PORT"]
+    MAIL_SUPPRESS_SEND = os.environ["MAIL_SUPPRESS_SEND"].lower() == "true"
```

Comment:
Do you really translate boolean to string here? :D Not the critique => just curious :)

Reply:
quite the opposite - I translate string (from env var) to boolean. It's a cheap way of doing that but works nicely. Env variables don't support types other than string, so that's why I'm doing it.